_posts/2023-08-30-release-2.6.0.md

```diff
@@ -0,0 +1,124 @@
+   This also avoids potentially [incorrect floating point results][]. (PR #4220)
+ - DATAWriter, CRD, PQR, and PDBQT files can now be correctly written to
+   compressed files. Before this change, any attempt to write to a compressed
+   format (gzip or bzip2) would lead to writing out an uncompressed file. (PR #4163)
+ - Prevent accidental merging of bond/angle/dihedral types when they are
+   defined as LAMMPS style string integers instead of tuples. This was leading
+   to an incorrect number of bond/angle/dihedral types being written to lammps
+   data files. (PR #4003)
+
+#### Enhancements:
+ - An `exclude_same` argument has been added to `InterRDF` allowing pairs of
+   atoms that share the same residue, segment or chain to be excluded from the
+   calculation. (PR #4161)
+ - LAMMPS reader now supports the `continuous` ChainReader option. (PR #4170)
+ - AtomGroup representation now returns atom indices in the same order
+   as they are stored in the AtomGroup. (PR #4191)
+
+#### Changes:
+ - As per NEP29, the minimum supported runtime version of NumPy has
+   been increased to 1.22.3. (PR #4160)
+ - The GSD package is now an optional dependency. (PR #4174)
+ - The MDAnalysis package now only supports GSD versions 3.0.1 or
+   above. (PR #4153)
+ - MDAnalysis no longer officially supports 32 bit installations,
+   (they are no longer tested in our continuous integration pipelines).
+   Note: no code changes have been made to disable 32 bit, although it
+   is known that new versions of most MDAnalysis core dependencies no longer
+   release 32 bit compatible versions. (PR #4176)
+ - The package license has been updated to [GPLv3+](https://www.gnu.org/licenses/gpl-3.0.en.html)
+   to better reflect the compatibility of GPLv2+ with Apache and GPLv3 licensed
+   codes. Additionally all new contributions from commit [44733fc](https://github.com/MDAnalysis/mdanalysis/commit/44733fc214dcfdcc2b7cb3e3705258781bb491bd)
+   onwards are made under the [LGPLv2.1+ license](https://www.gnu.org/licenses/old-licenses/lgpl-2.1.en.html). (PR #4219)
+
+#### Deprecations:
+ - The misspelt `Boltzman_constant` entry in `MDAnalysis.units` is now
+   deprecated in favour the correctly spelt `Boltzmann_constant`. (PRs #4230 and #4214)
+ - `MDAnalysis.analysis.hole2` is now deprecated in favour of a new
+   [HOLE2 MDAKit](https://www.mdanalysis.org/hole2-mdakit/). (PR #4200)
+
+## Author statistics
+
+This release was the work of 10 contributors, **4** of which are **new contributors**.
+
+Our **new contributors** are:
+
+@MohitKumar020291
+@Shubx10
+@ztimol
+@pillose
+
+## Acknowledgements
+
+MDAnalysis thanks [NumFOCUS][] for its continued support as our fiscal sponsor and the [Chan Zuckerberg Initiative][] for supporting MDAnalysis under an EOSS4 award.
+
+— The MDAnalysis Team
+
+
+[x86-64]: https://en.wikipedia.org/wiki/X86-64
+[aarch64]: https://en.wikipedia.org/wiki/AArch64
+[ppc64le]: https://en.wikipedia.org/wiki/Ppc64
```

Comment:
Apologies for my ignorance here, but we have a link here for ppc64 but it's not used in the post itself (though is mentioned in previous release posts) - is this intentional? (Not a huge thing and wouldn't let this stop releasing the blog post, but just wanted to check)

Reply:
Yeah we should just remove the link - so we used to occasionally test on ppc64le, and could claim that it was supported, but we fully lost access to Travis CI lately and can't test it. We do release conda-forge wheels for it still, we just don't really know if it works.

Reply:
ok - thanks! I'll remove it just to keep things cleaner